modules/netdevadd.php

```diff
@@ -220,10 +220,8 @@
         if (!$netdev['ownerid']) {
             $netdev['ownerid'] = '';
         }
-        $SMARTY->assign('netdev', $netdev);
-    } else {
-        $SMARTY->assign('netdev', array());
     }
+    $SMARTY->assign('netdev', array());
```

Comment:
To również wprowadza regresję względem dzisiejszej poprawki.

Reply:
https://github.com/chilek/lms/pull/2273/commits/bdbef66110baa2ded35aa27ed0584c609baf7f4b